packages/web/src/components/track/mobile/BottomButtons.tsx

```diff
@@ -93,16 +95,34 @@ const BottomButtons = (props: BottomButtonsProps) => {
 
   if (props.isUnlisted) {
     return (
-      <div className={styles.bottomButtons}>
-        <div className={styles.actions}>{shareButton}</div>
+      <Flex
+        ph='s'
+        pv='xs'
+        direction='row'
+        alignItems='center'
+        justifyContent='flex-end'
+        borderTop='default'
+      >
         {moreButton}
-      </div>
+      </Flex>
     )
   }
 
   return (
-    <div className={styles.bottomButtons}>
-      <div className={styles.actions}>
+    <Flex
+      ph={props.isTrack ? undefined : 's'}
```

Comment:
bit weird but 🤷 
i guess we can't remove the `isTrack` check for mobile-web bc we actually use the same `BottomRow` for both playlist and track tile, unlike desktop?